jwt/claims.go

```diff
@@ -110,6 +110,16 @@ func (s *Audience) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// MarshalJSON converts audience to json representation.
+func (s Audience) MarshalJSON() ([]byte, error) {
+	type Alias Audience
+	if len(s) == 1 {
+		return json.Marshal(s[0])
+	}
+	return json.Marshal((Alias)(s))
```

Comment:
Since audience is already a type alias for list of string, I believe we can call into the "original" MarshalJSON here without using the type alias and just casting to []string instead.

Reply:
Like this: https://play.golang.org/p/rhQ8o6hbFJd

Reply:
Thanks! Will give it a go tomorrow. This is my first Go go, so all feedback is welcome.